packages/react-theme/src/global/colors.ts

```diff
@@ -0,0 +1,877 @@
+import { GlobalSharedColors, ColorVariants, ProductBrandColors } from '../types';
+
+// Global Colors
+// https://www.figma.com/file/KB9oUjMKen2cKnyPG7RgdS/Design-tokens-superset?node-id=1297%3A14745
+
+export const grey = {
```

Comment:
Let's add a TODO to export these things as separate variables, otherwise they will not be tree shakable